src/script/builder/tir/for_frame.h

```diff
@@ -59,12 +59,16 @@ class ForFrame : public TIRFrame {
   TVM_DEFINE_MUTABLE_NOTNULLABLE_OBJECT_REF_METHODS(ForFrame, TIRFrame, ForFrameNode);
 };
 
-ForFrame Serial(PrimExpr start, PrimExpr stop, Map<String, ObjectRef> annotations);
-ForFrame Parallel(PrimExpr start, PrimExpr stop, Map<String, ObjectRef> annotations);
-ForFrame Vectorized(PrimExpr start, PrimExpr stop, Map<String, ObjectRef> annotations);
-ForFrame Unroll(PrimExpr start, PrimExpr stop, Map<String, ObjectRef> annotations);
+ForFrame Serial(PrimExpr start, PrimExpr stop,
+                Map<String, ObjectRef> annotations = Map<String, ObjectRef>());
```

Comment:
To be consistent with python API

```suggestion
ForFrame Serial(PrimExpr start, PrimExpr stop,
                Optional<Map<String, ObjectRef>> annotations = NullOpt);
```